### PR TITLE
Fix histodb forecastdiff

### PR DESCRIPTION
**Pull request by CBiasuzzi**

### CODE REVIEW

File: histoDB/histodb-dataserver/pom.xml [hunk 1]
```diff
@@ -137,6 +137,7 @@
         <dependency>
             <groupId>eu.itesla_project</groupId>
             <artifactId>cim1-import</artifactId>
+            <version>0.1-SNAPSHOT</version>
```
Comment (mathbagu) [inline histoDB/histodb-dataserver/pom.xml]: The version of ipst-entsoe dependencies must be declared in the parent pom.xml.

Comment (CBiasuzzi) [inline histoDB/histodb-dataserver/pom.xml]: any reason why the other references to hard-coded iidm-*  versions, in this very same pom.xml were not fixed during the idm-xxxx format merge?

Comment (mathbagu) [inline histoDB/histodb-dataserver/pom.xml]: The reason is "I didn't clean the dependencies yet"

Comment (CBiasuzzi) [inline histoDB/histodb-dataserver/pom.xml]: Anyway, in the latest commit cim-import  and iidm-xxxx version infos are now declared in histodb's top pom.xml

Comment (mathbagu) [inline histoDB/histodb-dataserver/pom.xml]: I didn't notice that histoDB was not a child of ipst parent pom.xml. Is there a reason?

**Pull request merged**